### DISCUSSION TITLE
Scaleway datacenters

### DISCUSSION THREAD

**Discussion by BarnabeD** [VarunAgw/CloudPingTest#34] [Ideas] [answered]
<table>
<tr>
 <td>Region
 <td>City
 <td>Country
 <td> Data-center
<tr>
 <td>PAR
 <td>Paris
 <td>France
 <td>DC-1
<tr>
 <td>PAR
 <td>Paris
 <td>France
 <td>DC-2
<tr>
 <td>PAR
 <td>Paris
 <td>France
 <td>DC-3
<tr>
 <td>AMS
 <td>Amsterdam
 <td>Amsterdam, Netherlands
 <td>DC-1
<tr>
 <td>AMS
 <td>Amsterdam
 <td>Amsterdam, Netherlands
 <td>DC-2
<tr>
 <td>AMS
 <td>Amsterdam
 <td>Amsterdam, Netherlands
 <td>DC-3
<tr>
 <td>WAW
 <td>Warsaw
 <td>Poland
 <td>DC-1
<tr>
 <td>WAW
 <td>Warsaw
 <td>Poland
 <td>DC-2
<tr>
 <td>WAW
 <td>Warsaw
 <td>Poland
 <td>DC-3
</table>

https://www.scaleway.com

Thanks for your work ! 👍

**Comment by VarunAgw** [comment, marked as answer]
Added
https://cloudpingtest.com/scaleway
